### PR TITLE
953327218 - AAUser on the design system, I am able to touch a dropdown field

### PR DESCRIPTION
**Pull request by elio-stalteri**

### CODE REVIEW

File: src/components/organism/Formik/FormikSelect/FormikSelect.js [hunk 1]
```diff
@@ -37,8 +36,9 @@ export class CustomSelect extends React.PureComponent {
 
   render() {
     const { name, id, caption } = this.props
-    const { errors } = this.props.formik
-
+    const { errors, touched } = this.props.formik
+    const error = getIn(errors, name)
```
Comment (frank-shenton) [inline src/components/organism/Formik/FormikSelect/FormikSelect.js:40]: Naming suggestions:
const errorText/Value
const isTouched

Took me a sec to differentiate error from errors and touch from touched.

Not a biggie though!

File: src/components/organism/Formik/FormikSelect/FormikSelect.test.js [hunk 2]
```diff
@@ -96,6 +98,7 @@ describe("CustomSelect Component Test", () => {
     const newFormikErrorProps = {
       values: { name: "" },
       errors: {},
+      touched: { name: false },
```
Comment (frank-shenton) [inline src/components/organism/Formik/FormikSelect/FormikSelect.test.js:101]: Do we have test for when touched and errors are both `true` and the `isError` prop is actually given?

File: src/components/organism/Formik/FormikSelect/FormikSelect.js (hunk 1 above)
Comment (elio-stalteri) [reply src/components/organism/Formik/FormikSelect/FormikSelect.js:40]: this is the suggested way from the Formik docs https://formik.org/docs/api/connect

File: src/components/organism/Formik/FormikSelect/FormikSelect.test.js (hunk 2 above)
Comment (elio-stalteri) [reply src/components/organism/Formik/FormikSelect/FormikSelect.test.js:101]: isError is being set by touched true and the following to this code is the assertion that isError is true

File: src/components/organism/Formik/FormikSelect/FormikSelect.js (hunk 1 above)
Comment (frank-shenton) [reply src/components/organism/Formik/FormikSelect/FormikSelect.js:40]: fair enough!

File: src/components/organism/Formik/FormikSelect/FormikSelect.test.js (hunk 2 above)
Comment (elio-stalteri) [reply src/components/organism/Formik/FormikSelect/FormikSelect.test.js:101]: image

Comment (frank-shenton) [reply src/components/organism/Formik/FormikSelect/FormikSelect.test.js:101]: Cool thanks, i see it now :)

**Review by frank-shenton — APPROVED** [review]

**Pull request merged**